force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -44,7 +51,13 @@ public with sharing class VariablesDataTypesOperators {
      * @return The difference between the two numbers.
      */
     public static Integer subtraction(Integer a, Integer b) {
-        return null; // Replace null with the variable you used to store the result
+        Integer value = 0; 
+        if (a != null && b != null){
+         value = a-b;
```

Comment:
Be sure to consistently add space around the operators for easier readability. `value = a - b;`